modules/ROOT/pages/introduction/cypher_aura.adoc

```diff
@@ -44,13 +44,8 @@ The Cypher Manual uses two different labels to differentiate this distinction:
 | label:aura-db-enterprise[AuraDB Enterprise] | Cypher feature only available on AuraDB Enterprise.
 |===
 
-////
-TODO: remove comment blocks once Aura Cheat Sheet has been published. 
-
 == Aura and the Cypher Cheat Sheet
 
 Each different tier of Aura has a customized version of the Cypher Cheat Sheet which only shows the features of Cypher available for the chosen tier. 
 
-The Aura Cheat Sheet can be accessed here: //Add url when available
-Note that the default tier is AuraDB Enterprise. 
-////
+The Cypher Cheat Sheet can be accessed link:{neo4j-docs-base-uri}/cypher-cheat-sheet/{page-version}/auradb-enterprise/[here]. Note that the default tier is AuraDB Enterprise. 
```

Comment:
While somewhat obvious it would be worth a mention on how you can select the version and tier in order to change the results returned.

Reply:
Do you not think the previous sentence ("Each different tier of Aura has a customized version of the Cypher Cheat Sheet which only shows the features of Cypher available for the chosen tier") makes that evident?

If not, maybe the following change would make it clearer:

"Each different tier of Aura has a customized version of the Cypher Cheat Sheet which, _when selected_, only shows the features of Cypher available for the chosen tier.